apps/dashboard/app/views/batch_connect/sessions/connections/_neither_stage_nor_submit.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= errors[key].each do |m| %>
+	<h4><%= m %></h4>

```

Comment:
This file appears to be using tabs instead of spaces for whitespace. Please fix.